services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_all_classrooms_progress_report.tsx

```diff
@@ -103,9 +103,12 @@ export default class StandardsAllClassroomsProgressReport extends React.Componen
         accessor: 'standard_name',
         sortType: sortTableByStandardLevel,
         resizable: false,
-        minWidth: 300,
-        Cell: ({row}) => this.renderTooltipRow(row),
-        style: {overflow: 'visible'},
+        minWidth: 500,
```

Comment:
Are headerWidth and minWidth 500 coupled? If so, I would extract as a constant to make that dependency explicit.

Also, I noticed this one is minWidth but previous examples were maxWidth, but I might just be noticing a pattern that isn't there.